hack/tagger/go.mod

```diff
@@ -4,18 +4,4 @@ go 1.16
 
 require github.com/google/go-containerregistry v0.6.0
 
-require (
```

Comment:
@stmcginnis can we revert go.sum modifications to this package. While I'm sure they're correct, we should prevent this tree from being mutated by this PR.